In [9]:
from slingshot.petition.layers.models import ContratoServicos, InversaoOnusProva
from slingshot.petition.layers.models import CobrancaIndevida, ImpossivelCancelar, DanoMoral, RegistroSerasa
from slingshot.petition.layers.parts import *
from slingshot.petition.core import Petition

In [10]:
pet = Petition()

In [11]:
pet.add(Author(name="Pi",
               profession="animal que fez esta porra",
               address="Rua do Pi",
               cpf="438.096.209-98"))

pet.add(ContratoServicos(service="adesao a plano de celular",
                        docs = ["contas", "reclamacoes"]))

pet.add(Claro_SP_re)

pet.add(InversaoOnusProva())

pet.add(CobrancaIndevida(tarifas_cobradas=["Dados a mais", "manutencao do chip"],
                         valor=2000,
                         docs=["contas"]))

pet.add(RegistroSerasa(divida_registrada="Conta do celular X",
                       valor=2000,
                        docs=["documentos anexos"]))

pet.add(ImpossivelCancelar(dano_moral=3000,
                           docs=["pedido de cancelamento", "email", "protocolo"]))

pet.add(DanoMoral(2000))

In [12]:
pet.compile()

In [13]:
pet.summary()

Acao condenatoria com base no CDC
#############
CAMADAS DA PETICAO:
Autor
Parte Re
Contexto - Prestacao de servicos
Preliminar - Inversao onus da prova
Cobranca indevida
Registro indevido no Serasa
Impossivel cancelar plano
Dano moral puro
#############
Valor da causa: R$ 11000


In [14]:
pet.print_text()

### **EXMO SR JUIZ DE DIREITO DO JEC DO TJSP**



Pi, animal que fez esta porra, domiciliado em Rua do Pi, sob o CPF 438.096.209-98 vem diante da Vossa Exa. ajuizar
Acao condenatoria com base no CDC
Em face de Claro S/A, com sede em Rua Henri Durant, 780, Santo Amaro, Sao Paulo - SP, CEP: 04709-110, sob o CNPJ 40.432.544/0001-47, pelas razoes de fato e Direito aqui expostas:

#### **PRELIMINARES**
**1.** Em virtude da nitida situacao de hipossuficiente da parte autora, e do art 6o, inciso VIII do CDC e, faz-se necessariainversao do onus da prova em favor da parte autora neste processo.
* *Art. 6°. São direitos básicos do consumidor:* 
* *(...) VIII – a facilitação da defesa de seus direitos, inclusive com a inversão do ônus da prova, a seu favor, no processo civil, quando, a critério do juiz, for verossímil a alegação ou quando for ele hipossuficiente, segundo as regras ordinárias de experiências (...).*


#### **DOS FATOS**
**2.** Conforme da documentacao anexa ('contas', 'reclamacoes

### Interpretando markdown

### **EXMO SR JUIZ DE DIREITO DO JEC DO TJSP**



Pi, animal que fez esta porra, domiciliado em Rua do Pi, sob o CPF 438.096.209-98 vem diante da Vossa Exa. ajuizar
Acao condenatoria com base no CDC
Em face de Claro S/A, com sede em Rua Henri Durant, 780, Santo Amaro, Sao Paulo - SP, CEP: 04709-110, sob o CNPJ 40.432.544/0001-47, pelas razoes de fato e Direito aqui expostas:

#### **PRELIMINARES**
**1.** Em virtude da nitida situacao de hipossuficiente da parte autora, e do art 6o, inciso VIII do CDC e, faz-se necessariainversao do onus da prova em favor da parte autora neste processo.
* *Art. 6°. São direitos básicos do consumidor:* 
* *(...) VIII – a facilitação da defesa de seus direitos, inclusive com a inversão do ônus da prova, a seu favor, no processo civil, quando, a critério do juiz, for verossímil a alegação ou quando for ele hipossuficiente, segundo as regras ordinárias de experiências (...).*


#### **DOS FATOS**
**2.** Conforme da documentacao anexa ('contas', 'reclamacoes') parte autora firmou com a parte requerida contrato de prestacao de servicos, mais especificamente por meio de adesao a plano de celular. Ocorre que, durante a execucao do contrato, a re teve uma serie de condutas que passaram a prejudicar a autora.

**3.** Ainda conforme a documentacao, a parte autora tentou de todos os meios amigaveis possiveispara a solucao dos problemas que seguem, tornando esta acao a unica solucao possivel.

**4.** Conforme a documentacao anexa 'contas', a parte re cobrou indevidamente da autora valores que perfazezm o montate de R$ 2000, sob a invalida justificativa  de corresponderem a ['Dados a mais', 'manutencao do chip'].

**5.** A parte autora ressalta que jamais anuiu com a cobranca indevida desses valores, seja como prestacao de servicos adicionais, reajuste no plano ou qualquer outro tipo de cobranca, de maneira que resta ilicita e incorreta a referida cobranca de valores.

**6.** Conforme a documentacao anexa ('documentos anexos'), a parte re registrou divida inexistente da autora no valor o montate de R$ 2000, nas empresas de score de credito, a titulo de Conta do celular X. Ocorre que as referidas dividas nao existem

**7.** A parte autora ainda ressalta nao ser devedor contumaz, de forma que o registro indevido do nome nas empresas de score de credito caracteriza dano moral.

**8.** Ocorre que a parte autora vem tentando, sem sucesso, cancelar seu plano e, depois de varias tentativas, nao conseguiu ter sucesso. Isso se deu por conduta ativa e dolosa da parte re que, desnecessariamente demora excessivamente e dificulta toda forma de comunicacaono que diz respeito ao cancelamento do plano.

**9.** Vale dizer que a parte re sempre mostrou prontidao quando do momento da contratacao, mas quando se quis proceder o cancelamento, passou a agir com essa conduta desrespeitosa que causou grande desconforto e sentimento de humilhacao, traduzido em dano moral, para a parte autora.

**10.** Isso encontra respaldo documental na documentacao anexa ('pedido de cancelamento', 'email', 'protocolo')


#### **DOS DIREITO**
**11.** De acordo com o Art 42, Parágrafo Único, do CDC, quando houver cobranca indevida, o consumidor tem direito a repeticao do indebito em dobro (*Parágrafo único. O consumidor cobrado em quantia indevida tem direito à repetição do indébito, por valor igual ao dobro do que pagou em excesso, acrescido de correção monetária e juros legais, salvo hipótese de engano justificável.*)

**12.** O mesmo ainda é encontra base na jurisprudência, que reconhece, inclusive, a exitência de dano moral em caso de cobrança indevida - inclusive por causa da ocorrência de desvio produtivo do consumidor:
 * PRESTAÇÃO DE SERVIÇOS INEXISTENTE - COBRANÇA INDEVIDA - RESTITUIÇÃO EM DOBRO E COMPENSAÇÃOPOR DANO MORAL – RECONHECIMENTO – FIXAÇÃO EM R$ 7.000,00 - PRETENSÃO DA AUTORA EMMAJORAÇÃO – IMPERTINÊNCIA - RECURSO NÃO PROVIDO. - *Tribunal de Justiça de São Paulo TJ-SP - Apelação Cível : AC1000854-52.2018.8.26.0161*

**13.** De acordo com o Art 42, caput do CDC/CC, o consumidor não será exposto a ridículo quando da cobrança de dívidas. (*Art. 42. Na cobrança de débitos, o consumidor inadimplente não será exposto a ridículo, nem será submetido a qualquer tipo de constrangimento ou ameaça.*)

**14.** Ocorre que, neste caso, isso aconteceu, tendo em vista que a inserção de seu nome indevidamente no cadastro de devedores o expõe ao ridículo e abala a sua imagem, além de impedir que adquira crédito no mercado. Por isso, deve haver remoção imediata de seu nome dos respectivos cadastros e compensacao pelo dano moral e à sua imagem. 

**15.** Esse entendimento foi validado em jurisprudencia, que, inclusive, reconhece a existência de dano moralpara esse tipo de caso de inclusão indevida no cadastro de devedores do Serasa:
 * PRESTAÇÃO DE SERVIÇOS DE TELEFONIA AÇÃO DE CANCELAMENTO DE COBRANÇA INDEVIDA E DE REGISTRO INDEVIDO JUNTO AO SERASA/SPCC/C REPARAÇÃO DE DANOS MORAIS COM PEDIDO DE TUTELA ANTECIPADA INEXIGIBILIDADE DOS DÉBITOS SERVIÇO CONTRATADO FUNCIONAMENTO NÃO COMPROVADO PESSOA JURÍDICA NEGATIVAÇÃO INDEVIDA DO NOME DA AUTORA EM ÓRGÃOS DE PROTEÇÃO AO CRÉDITO DANO MORALCARACTERIZADO SENTENÇA QUE ARBITROU QUANTIA EM CONFORMIDADE COMOS CRITÉRIOS DE PROPORCIONALIDADE E RAZOABILIDADE INDENIZAÇÃO DEVIDA FIXADA EM R$ 10.000,00 VALOR MANTIDO SENTENÇA MANTIDA. *TJSP- Apelação cível AC :  0058685-32.2012.8.26.0114 publicado em 24/01/2017*

**16.** Ademais do dano moral, cuja necessidade de ressarcimento tem resguardo legislativono Codigo Civil art 186, 188 e 187, o mesmo, para esse tipo de caso do consumidor, encontra respaldo noart 14, caput do CDC (*Art. 14. O fornecedor de serviços responde, independentemente da existência de culpa, pela reparação dos danos causados aos consumidores por defeitos relativos à prestação dos serviços, bem como por informações insuficientes ou inadequadas sobre sua fruição e riscos.*)

**17.** Existe ainda respaldo jurisprudencial no sentido de entender a gravidade e a necessidade de ressarcimento deste tipo de dano, inclusive por causa do desvio produtivo do consumidor, como ja foi reconhecido pelo STJ:
* PROCESSO CIVIL E DIREITO DO CONSUMIDOR. RECURSO ESPECIAL. AÇÃO CIVIL PÚBLICA. NEGATIVA DE PRESTAÇÃO JURISDICIONAL. AUSÊNCIA. JUNTADA DE DOCUMENTOS COM A APELAÇÃO. POSSIBILIDADE. VÍCIO DO PRODUTO. REPARAÇÃO EM 30 DIAS. RESPONSABILIDADE OBJETIVA DO COMERCIANTE.1. Ação civil pública ajuizada em 07/01/2013, de que foi extraído o presente recurso especial, interposto em 08/06/2015 e concluso ao Gabinete em 25/08/2016. Julgamento pelo CPC/73.2. Cinge-se a controvérsia a decidir sobre: (i) a negativa de prestação jurisdicional (art. 535, II, do CPC/73); (ii) a preclusão operada quanto à produção de prova (arts. 462 e 517 do CPC/73); (iii) a responsabilidade do comerciante no que tange à disponibilização e prestação de serviço de assistência técnica (art. 18, caput e § 1º, do CDC).3. Devidamente analisadas e discutidas as questões de mérito, e fundamentado o acórdão recorrido, de modo a esgotar a prestação jurisdicional, não há que se falar em violação do art. 535, II, do CPC/73.4. Esta Corte admite a juntada de documentos, que não apenas os produzidos após a inicial e a contestação, inclusive na via recursal, desde que observado o contraditório e ausente a má-fé. **5. À frustração do consumidor de adquirir o bem com vício, não é razoável que se acrescente o desgaste para tentar resolver o problema ao qual ele não deu causa, o que, por certo, pode ser evitado – ou, ao menos, atenuado – se o próprio comerciante participar ativamente do processo de reparo, intermediando a relação entre consumidor e fabricante, inclusive porque, juntamente com este, tem o dever legal de garantir a adequação do produto oferecido ao consumo. 6. À luz do princípio da boa-fé objetiva, se a inserção no mercado do produto com vício traz em si, inevitavelmente, um gasto adicional para a cadeia de consumo, esse gasto deve ser tido como ínsito ao risco da atividade, e não pode, em nenhuma hipótese, ser suportado pelo consumidor. Incidência dos princípios que regem a política nacional das relações de consumo, em especial o da vulnerabilidade do consumidor (art. 4º, I, do CDC) e o da garantia de adequação, a cargo do fornecedor (art. 4º, V, do CDC), e observância do direito do consumidor de receber a efetiva reparação de danos patrimoniais sofridos por ele (art. 6º, VI, do CDC).** 7. Como a defesa do consumidor foi erigida a princípio geral da atividade econômica pelo art. 170, V, da Constituição Federal, é ele – consumidor – quem deve escolher a alternativa que lhe parece menos onerosa ou embaraçosa para exercer seu direito de ter sanado o vício em 30 dias – levar o produto ao comerciante, à assistência técnica ou diretamente ao fabricante –, não cabendo ao fornecedor impor-lhe a opção que mais convém. 8. Recurso especial desprovido. *RESP - 1634851, Rel. Min. Nancy Andrighi*


#### **DOS PEDIDOS**
**1.** Seja deferida a inversao do onus da prova em favor da parte autora.

**2.** A repeticao do indebito em dobro, de acordo com o exposto e no valor de R$ 4000.

**3.** A imediata retirada da anotacao nas empresas de Score de credito da alegada dividaindevidamente cobrada pela parte re

**4.** O cancelamento imediato de todos os servicos que a parte re se recusou a cancelar ou dificultou o cancelamento.

**5.** A condenacao da parte re ao pagamento de um total de R$ 5000 a titulo de danos morais, em virtude do exposto

**6.** Da-se a causa o valor de R$ 11000


Termos em que pede deferimento
Pi

In [16]:
pet1 = Petition()
print("Vamos processar a TIM? ")
name = input("Quem e vc? ")
profession = input("Qual sua profissao? ")
address = input("Qual seu endereco? ")
cpf = input("Qual seu cpf? ")
pet1.add(Author(name=name,
                profession=profession,
               address=address,
               cpf=cpf))

pet1.add(ContratoServicos(service="adesao a plano de celular",
                        docs = ["contas", "reclamacoes"]))

pet1.add(InversaoOnusProva())

if input("Teve cobranca indeivida? ") == "sim":
    tarifas = [input("Quais tarifas te cobraram a mais? ")]
    valor = int(input("E quanto eles te cobraram? "))
    docs = [input("Agora diga quais documentos voce tem que provam isso ai...")]
    pet1.add(CobrancaIndevida(tarifas_cobradas=tarifas,
                         valor=valor,
                         docs=docs))
if input("Ta dificil cancelar o plano tambem?") == "sim":
    docs2 = [input("Me diga os documentos que vai usar pra provar isso ")]
    valor2 = int(input("E quanto quer de dano moral pra compensar isso ai? "))
    pet1.add(ImpossivelCancelar(dano_moral=3000,
                           docs=["pedido de cancelamento", "email", "protocolo"]))
    
pet1.compile()
    
print("Partiu processar a TIM!!!")
pet1.print_text()

Vamos processar a TIM? 
Quem e vc? Tris
Qual sua profissao? Estudante
Qual seu endereco? Rua Benis 35
Qual seu cpf? 123.123.123-12
Teve cobranca indeivida? sim
Quais tarifas te cobraram a mais? Chip, dados a mais
E quanto eles te cobraram? 3000
Agora diga quais documentos voce tem que provam isso ai...contas
Ta dificil cancelar o plano tambem?nao
Partiu processar a TIM!!!
EXMO SR JUIZ DE DIREITO DO JEC DO TJSP



Tris, Estudante, domiciliado em Rua Benis 35, sob o CPF 123.123.123-12 vem diante da Vossa Exa. ajuizar
Acao condenatoria com base no CDC
PRELIMINARES
1. Em virtude da nitida situacao de hipossuficiente da parte autora, do art XXXX do CDC e art XXX do CPC e CC e dos julgados XXXX, faz-se necessariainversao do onus da prova em favor da parte autora neste processo.

DOS FATOS
2. Conforme da documentacao anexa ('contas', 'reclamacoes') parte autora firmou com a parte requerida contrato de prestacao de servicos, mais especificamente por meio de adesao a plano de celular. Ocorre qu